experimental/generation/generator/src/library/dialogGenerator.ts

```diff
@@ -78,7 +78,7 @@ export async function isUnchanged(path: string): Promise<boolean> {
     return result
 }
 
-async function writeFile(path: string, val: any, feedback: Feedback) {
+export async function writeFile(path: string, val: any, feedback: Feedback) {
```

Comment:
>writeFile [](start = 22, length = 9)

do you use this function?

Reply:
Yes, I use this function in integration.ts